src/phasorpy/phasor.py

```diff
@@ -0,0 +1,514 @@
+
+    Examples
+    --------
+    Phasor coordinates of a single lifetime component at 80 MHz:
+
+    >>> phasor_from_lifetime(80.0, 1.9894368)  # doctest: +NUMBER
+    (0.5, 0.5)
+
+    Phasor coordinates of two lifetime components with equal fractional
+    intensities:
+
+    >>> phasor_from_lifetime(
+    ...     80.0, [3.9788735, 0.9947183], [0.5, 0.5]
+    ... )  # doctest: +NUMBER
+    (0.5, 0.4)
+
+    Phasor coordinates of a double-exponential decay with equal
+    pre-exponential amplitudes:
+
+    >>> phasor_from_lifetime(
+    ...     80.0, [3.9788735, 0.9947183], [0.5, 0.5], is_preexp=True
+    ... )  # doctest: +NUMBER
+    (0.32, 0.4)
+
+    """
+    tau = numpy.array(lifetime, dtype=numpy.float64, copy=True)
+    if tau.ndim > 1:
+        raise ValueError('lifetime must be scalar or one-dimensional array')
+    if fraction is None:
+        frac = numpy.ones_like(tau)
+    else:
+        frac = numpy.array(fraction, dtype=numpy.float64, copy=True)
+        if tau.shape != frac.shape:
+            raise ValueError(f'shape mismatch {tau.shape} != {frac.shape}')
+    if is_preexp:
+        # preexponential amplitudes to fractional intensities
+        frac *= tau
+    frac /= numpy.sum(frac)  # TODO: check for zero
+    tau *= frequency * 2e-3 * math.pi  # omega_tau
+    tmp = numpy.square(tau)
+    tmp += 1.0
+    tmp **= -1
+    tmp *= frac
+    re = numpy.sum(tmp)
+    tmp *= tau
+    im = numpy.sum(tmp)
+    return re, im
+
+
+def phasor_center(
+    real: ArrayLike,
+    imag: ArrayLike,
+    /,
+    *,
+    skip_axes: tuple[int, ...] | None = None,
+    method: Literal['mean', 'median'] = 'mean',
+) -> tuple[NDArray[Any], NDArray[Any]]:
+    """Return center of phasor coordinates.
+
+    Parameters
+    ----------
+    real : array_like
+        Real component of the phasor coordinates.
+    imag : array_like
+        Imaginary component of the phasor coordinates.
+    skip_axes : tuple of int, optional
+        Axes to be excluded during center calculation. If None, all
+        axes are considered.
+    method : str, optional
+        Method used for center calculation:
+            - 'mean': Arithmetic mean of the coordinates.
+            - 'median': Spatial median of the coordinates.
+
+    Returns
+    -------
+    real_center: ndarray
+        Real center coordinates calculated based on the specified method.
+    imag_center: ndarray
+        Imaginary center coordinates calculated based on the specified method.
+
+    Raises
+    ------
+    ValueError
+        If the specified method is not supported.
+        If the shapes of the 'real' and 'imag' do not match.
+
+    Examples
+    --------
+    Compute center coordinates with the 'mean' method:
+    >>> phasor_center([1.0, 2.0, 3.0], [4.0, 5.0, 6.0], method='mean')
+    (2.0, 5.0)
+
+    Compute center coordinates with the 'median' method:
+    >>> phasor_center([1.0, 2.0, 3.0], [4.0, 5.0, 6.0], method='median')
+    (2.0, 5.0)
```

Comment:
These examples still don't render correctly. Add an empty line between text and code.